shell/platform/android/android_context_gl.h

```diff
@@ -16,17 +16,18 @@
 namespace shell {
 
 class AndroidContextGL : public ftl::RefCountedThreadSafe<AndroidContextGL> {
+  FRIEND_MAKE_REF_COUNTED(AndroidContextGL);
```

Comment:
Done. Sorry. I though putting it at the top better conveyed that this instance could only be `MakeRefCounted`ed. But that does not seem to the convention. Fixed.